aiohttp/client_reqrep.py

```diff
@@ -33,6 +34,10 @@
 __all__ = ('ClientRequest', 'ClientResponse', 'RequestInfo')
 
 
+ContentDisposition = collections.namedtuple(
+    'ContentDisposition', ('value', 'parameters', 'filename'))
```

Comment:
I think `value` is not the best choice. The [RFC](https://tools.ietf.org/html/rfc6266#section-4.1) defines it as a `type` which suites more to what the value it holds.

Reply:
Agree